torch/_torch_docs.py

```diff
@@ -2220,10 +2218,63 @@ def merge_dicts(*dicts):
 
 Example::
 
-    >>> torch.conj(torch.tensor([-1 + 1j, -2 + 2j, 3 - 3j]))
+    >>> torch.conj_physical(torch.tensor([-1 + 1j, -2 + 2j, 3 - 3j]))
     tensor([-1 - 1j, -2 - 2j, 3 + 3j])
 """.format(**common_args))
 
+add_docstr(torch.conj,
+           r"""
+conj(input) -> Tensor
```

Comment:
`conj_physical()` is interesting; how do you think of `conj_physical()` vs. extending the signature of `conj()` to `conj(input, *, as_view=True)`?

Reply:
We talked about this in the past; I think it's simpler to have `conj_physical` be a separate function as `conj_physical` is a traditional structured kernel with inplace/outplace variants, where as the kwarg-ed conj would be some bastardly hybrid depending on the kwarg setting. Additionally, the `as_view` argument goops up an otherwise simple signature. WDYT

Reply:
Sure; I suppose if we were being pedantic or didn't have BC-concerns we'd just have the names `conj` for `conj_physical` and `conj_view` for `conj`, then?

Reply:
Opposite. We're purposely breaking BC by turning `conj` into `conj_view`. The point is that 99% this is what a typical end user wants (this is what boedekker argued in the past), so we are giving the correct defaults.